XCUITests/AuthenticationTest.swift

```diff
@@ -341,13 +339,14 @@ class AuthenticationTest: BaseTestCase {
         typePasscode("1337")
         waitforExistence(app.staticTexts["Immediately"])
         app.staticTexts["After 15 minutes"].tap()
-
+        
         navigator.goto(PasscodeSettings)
         waitforExistence(app.tables["AuthenticationManager.settingsTableView"].staticTexts["After 15 minutes"])
-
+        
```

Comment:
Ok; Could you do two things: in Xcode, got to Preferences --> Text Editing, and select the two options:

 * Automatically trim whitespace
 * Include whitespace-only lines

Second, and you might not need to do this now: go through each of the files in your this PR and find the trailing whitespace lines. This might be as easy as opening them and then saving them again. 

The reason we like this is that these whitespace diffs just add noise to the patch.

I hope this helps.

Reply:
ah, i was looking for it and 'nclude whitespace-only lines' wasn't checked. also I 'swiftlint autocorrect' ed to fix the whitespaces